frontend/src/metabase/components/Popover/Popover.module.css

```diff
@@ -261,25 +166,3 @@
   height: 6px;
   pointer-events: none;
 }
-
-/* transition classes */
-
-:global(.Popover-appear),
```

Comment:
Transition classes seem to be unused. I try opening dashboard filter, and there is already no transition.